src/main/java/net/sf/jabref/gui/preftabs/AdvancedTab.java

```diff
@@ -61,6 +77,39 @@ public AdvancedTab(JabRefPreferences prefs) {
         useCaseKeeperOnSearch = new JCheckBox(Localization.lang("Add {} to specified title words on search to keep the correct case"));
         useUnitFormatterOnSearch = new JCheckBox(Localization.lang("Format units by adding non-breaking separators and keeping the correct case on search"));
 
+        defaultConsole = new JRadioButton(Localization.lang("Use default terminal emulator"));
+        specifiedConsole = new JRadioButton(Localization.lang("Specify terminal emulator:"));
+        consoleEmulatorPath = new JTextField(20);
```

Comment:
Why did you set the columns to 20? In my opinion you can use more as it seems there is still space to the right in the preference dialog.